environs.py

```diff
@@ -195,7 +195,10 @@ def _deserialize(self, value, *args, **kwargs) -> Path:
 
 
 class LogLevelField(ma.fields.Int):
-    def _format_num(self, value) -> int:
+    # Type ignore, because the return type annotation of the super
+    # class is a private TypeVar incompatible with int. The annotation
+    # in super should probably be Any.
```

Comment:
Good point; would you like to send a PR to marshmallow? =)

Reply:
Can definitely do this!